contracts/neutron_interchain_queries/src/contract.rs

```diff
@@ -113,13 +113,129 @@ pub fn execute(
     }
 }
 
+pub fn register_balance_query(
+    deps: DepsMut<InterchainQueries>,
+    env: Env,
+    connection_id: String,
+    addr: String,
+    denom: String,
+    update_period: u64,
+) -> NeutronResult<Response<NeutronMsg>> {
+    let msg = register_balance_query_msg(
+        deps,
+        env,
+        connection_id.clone(),
+        addr.clone(),
+        denom.clone(),
+        update_period,
+    )?;
+
+    let attrs: Vec<Attribute> = vec![
```

Comment:
https://github.com/neutron-org/neutron/pull/49/files this pr adds similar events for create / update / remove operations, not sure we should duplicate them here